### PR TITLE
feat: v1.0.0 release

### PR DESCRIPTION
**Pull request by nschonni**
- Cleanup old deps
- Remove lock to prevent need to releases for unrelated sub-dependency security issues
- Bump to 1.0.0 since this is effectly "done" as a library that probably won't change further
- Use the latest nod-sass release
- Remove mkdirp for native calls
- Set minimum Node to 12, to match upstream

### CODE REVIEW

**Comment by seanmakesgames** [comment]
Good on y'all for getting a 1.0 prepped here! Looking forward to the security patches. :D

**Comment by nschonni** [comment]
@am11 thanks for tagging the release, it looks like it successfully published 1.0.0 even though the tag name is wrong 😉

**Comment by am11** [comment]
Ah, thanks for spotting it, I missed that. 🙈 We can re-tag it as `1.0.1` (after #151) to avoid conflicts. I created the tag via GH UI (create release -> new tag).

Perhaps it would be nice if the build fails in this case. 🤔

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
```
[the remark this answers is not in the document]
Comment (am11) [reply .github/workflows/publish.yml:8]: Apparently, this (`created`) event is not fired if we Draft a release then publish it. It is only fired when either release was directly published or release was created from the tag.
(I posted a draft for v1.0.1 and noticed this behavior)

**Pull request merged**